larky/src/main/resources/stdlib/xml/etree/ElementTree.star

```diff
@@ -806,7 +806,15 @@ def _ElementTree(element=None, file=None):
                                     of start/end tags
 
         """
-        return Error("Unsupported write method")
+        # return Error("Unsupported write method")
```

Comment:
Just remove this comment

Reply:
I feel like it is better to keep it just in case some user mistakenly uses write method as official doc says. I add a new inner method _write for tostring to use